composer.json

```diff
@@ -37,27 +38,38 @@
         }
     },
     "scripts": {
+        "bootstrap-test-environment": [
+            "Tests/Application/bin/console doctrine:database:create --if-not-exists --env test",
+            "Tests/Application/bin/console doctrine:schema:update --force --env test"
+        ],
         "lint": [
-            "@lint-composer",
             "@phpstan",
-            "@lint-yaml",
-            "@lint-schema"
+            "@php-cs",
+            "@lint-container",
+            "@lint-composer",
+            "@lint-doctrine"
         ],
         "test": [
             "@phpunit"
         ],
-        "lint-composer": "@composer validate --strict",
-        "lint-yaml": "@php Tests/Application/bin/console lint:yaml Resources/config",
-        "lint-schema": "@php Tests/Application/bin/console doctrine:schema:validate",
+        "phpunit": "vendor/bin/phpunit",
         "phpstan": [
             "@php Tests/Application/bin/console cache:warmup --env dev",
             "@php vendor/bin/phpstan analyse"
         ],
-        "bootstrap-test-environment": [
-            "@php Tests/Application/bin/console doctrine:database:create --if-not-exists",
-            "@php Tests/Application/bin/console doctrine:schema:update --force"
+        "php-cs": "vendor/bin/php-cs-fixer fix --verbose --diff --dry-run",
+        "php-cs-fix": "vendor/bin/php-cs-fixer fix",
+        "lint-composer": "@composer validate --strict",
+        "lint-container": [
+            "Tests/Application/bin/console lint:container --env dev",
```

Comment:
@alexander-schranz the test setup in this bundle does not have a `adminconsole` and a `websiteconsole` and i failed to add them 🙈

Reply:
can you help me here or should we leave it like this?

Reply:
added